src/components/breadcrumbs/breadcrumbs.component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import tagComponent, { TagProps } from "../../__internal__/utils/helpers/tags";
+import StyledBreadcrumbs from "./breadcrumbs.style";
+import useLocale from "../../hooks/__internal__/useLocale";
+
+export interface BreadcrumbsProps extends TagProps {
+  /** Child crumbs to display */
+  children: React.ReactNode;
+}
+
+export const Breadcrumbs = React.forwardRef(
+  ({ children, ...rest }: BreadcrumbsProps) => {
+    const l = useLocale();
+    return (
+      <StyledBreadcrumbs
+        role="navigation"
+        {...tagComponent("breadcrumbs", rest)}
+        aria-label={l.breadcrumbs.ariaLabel()}
+      >
+        <ol>{children}</ol>
+      </StyledBreadcrumbs>
+    );
+  }
+);
+
```

Comment:
Similar to `Crumb`, since this has been defined as a forward ref we will need to specify `ref` on the root element like so:

```suggestion
export const Breadcrumbs = React.forwardRef<HTMLNavElement, BreadcrumbsProps>(
  ({ children, ...rest }: BreadcrumbsProps, ref) => {
    const l = useLocale();
    return (
      <StyledBreadcrumbs
        ref={ref}
        role="navigation"
        {...tagComponent("breadcrumbs", rest)}
        aria-label={l.breadcrumbs.ariaLabel()}
      >
        <ol>{children}</ol>
      </StyledBreadcrumbs>
    );
  }
);

```

Reply:
This is still missing a `ref`